### PR TITLE
Added teams handles and constants

### LINKED ISSUE #2: Add descriptions of library constants

### PR DESCRIPTION
**Pull request by gmegan**
Resolve #2

### CODE REVIEW

File: content/library_handles.tex [hunk 1]
```diff
@@ -13,6 +13,30 @@
 \tabularnewline \hline
 \endhead
 %%
+\LibHandleDecl{SHMEM\_TEAM\_WORLD} &
+Handle of type \CTYPE{shmem\_team\_t} that corresponds to the
+default team of all pes in the OpenSHMEM program.  All point-to-point
```
Comment (nspark) [inline content/library_handles.tex]: - `pes` &#8594; `\acp{PE}`
- `OpenSHMEM` &#8594; `\openshmem`

File: content/library_handles.tex [hunk 2]
```diff
@@ -13,6 +13,30 @@
...
+communication operations and synchronizations that do not specify a team
+are performed on the default team.
+See Section~\ref{subsec:team} for more detail about its use.
+\tabularnewline \hline
+%%
+\LibHandleDecl{SHMEM\_TEAM\_NODE} &
+Handle of type \CTYPE{shmem\_team\_t} that corresponds a team of pes
```
Comment (nspark) [inline content/library_handles.tex]: `pes` &#8594; `\acp{PE}`

File: content/library_handles.tex [hunk 3]
```diff
@@ -13,6 +13,30 @@
...
+which share node level resources, such as shared memory, network
+interfaces, etc. When this handle is used by some pe, it will refer
+to the node level team containing that pe.
```
Comment (nspark) [inline content/library_handles.tex]: `pe` &#8594; `\ac{PE}` (this and previous line)

File: content/library_handles.tex [hunk 4]
```diff
@@ -13,6 +13,30 @@
...
+See Section~\ref{subsec:team} for more detail about its use.
+\tabularnewline \hline
+%%
+\LibHandleDecl{SHMEM\_TEAM\_WORLD} &
```
Comment (nspark) [inline content/library_handles.tex]: Is `SHMEM_TEAM_WORLD` in here twice?

**Review by nspark — CHANGES_REQUESTED** [review]
Request editorial changes before merging.

One design/implementation point for discussion -- for which a resolution is not expected prior to merging -- is the extent to which a team created with the `SHMEM_TEAM_NOCOLLECTIVE` option is broadly useful. If `NOCOLLECTIVE` still allowed barriers, but would reduce memory usage (e.g., needed for internal equivalent `pWork` or `pSync` state), then I think this could definitely be useful.

File: content/library_handles.tex (hunk 4 above)
Comment (gmegan) [reply content/library_handles.tex]: Ah, copy pasta problem. That was meant to be SHMEM_TEAM_NULL but then that got added as a constant. I will remove this.

**Review by nspark — APPROVED** [review]

**Comment by nspark** [comment]
:+1: to merge

**Pull request merged**